lib/backend/compat/compat.go

```diff
@@ -144,6 +150,11 @@ func (c *ModelAdaptor) Apply(d *model.KVPair) (*model.KVPair, error) {
 			return d, nil
 		}
 	case model.NodeKey:
+		// Quick fix for k8s Node call.  The k8s Node.Apply call requires a complete model.Node object,
+		// therefore we do not need to break down the node object.
```

Comment:
Correct the KDD API will complain about not supporting those API calls with some no-op messages in the debug logs.